template/package.json

```diff
@@ -75,7 +75,7 @@
       "node"
     ],
     "transformIgnorePatterns": [
-      "node_modules/(?!react-native|@react-native|react-native-vector-icons)"
+      "node_modules/(?!react-native|@react-native|react-native-vector-icons|@react-navigation)"
```

Comment:
When you add the stack navigator, it pulls in some react-navigation items (an icon, specifically) that jest cannot tolerate as it attempts to process things. Adding react-navigation to jest transform ignore fixes `yarn test:all`